backend/src/Edna/ExperimentReader/Types.hs

```diff
@@ -4,12 +4,18 @@ module Edna.ExperimentReader.Types
   , ParameterType (..)
   , Signal (..)
   , CellType (..)
+  , PointYX (..)
   ) where
 
 import Universum
 
+import qualified GHC.Show as S
+
 import Codec.Xlsx (CellValue(..))
-import Fmt (Buildable(..))
+import Fmt (Buildable(..), (+|), (|+), pretty)
+
+data PointYX = PointYX (Int, Int)
```

Comment:
I would define it as
```
data PointYX = PointYX { pointY :: Int, pointX :: Int }
```

and then probably you can even drop `YX` from the name, but I don't insist on doing that, it's minor.

Reply:
I see that you changed it to be `newtype` and that `Codec.Xlsx` apparently takes position as `(Int, Int)`, so it's sort of justified.